### ISSUE TITLE
eslint@2.5.3 doesn't satisfy peerDependencies on CI

### ISSUE THREAD

**Issue by webPapaya** [kodyl/stilr#27]
I just added PR #26 and on Node 4.x TravicCI is complaining:

````
The package eslint@2.5.3 does not satisfy its siblings' peerDependencies requirements!
````

https://travis-ci.org/kodyl/stilr/builds/120002576

**Comment by danieljuhl** [comment]
Yes, I see that... it's probably also time to test with `react@15`, so I'll probably do a general dependency check.

**Comment by danieljuhl** [comment]
Fixed in `v1.2.3`
